src/CheckView.cpp

```diff
@@ -7,6 +8,7 @@
 #include <stdlib.h>
 
 #include "Account.h"
+#include "CalendarButton.h"
```

Comment:
CalendarButton.h is missing it seems

Reply:
Blast! Once more I forgot to git add the main character of the show...
PR amended.